shared/constants/first-party-contracts.ts

```diff
@@ -8,7 +8,10 @@ export const FIRST_PARTY_CONTRACT_NAMES: Record<string, Record<Hex, Hex>> = {
   'MetaMask Validator Staking': {
     [CHAIN_IDS.MAINNET]: '0xDc71aFFC862fceB6aD32BE58E098423A7727bEbd',
   },
-  'MetaMask Pool Staking': {
+  'MetaMask Pooled Staking': {
+    [CHAIN_IDS.MAINNET]: '0x4FEF9D741011476750A243aC70b9789a63dd47Df',
+  },
+  'Liquid Staking': {
```

Comment:
Should this also include the `MetaMask` prefix to be explicit that it's our contract?